spec/game_spec.rb

```diff
@@ -0,0 +1,115 @@
+require './lib/game.rb'
+require './lib/player.rb'
+
+RSpec.describe 'testing the game methods' do
+  let(:player1) { Player.new('Grace', 'x') }
+  let(:player2) { Player.new('Olamide', 'o') }
+  let(:game) { Game.new(player1, player2) }
+
+  it 'it chooses sign for players' do
+    arr = Game.choose_signs
+    expect(arr.sort).to eq(%w[x o].sort)
+  end
+
+  it 'it chooses sign for players' do
+    arr = Game.choose_signs
+    expect(arr.sort).not_to eq(%w[x k].sort)
+  end
```

Comment:
These tests should be wrapped in a `describe` block. Since both of them are testing the `self.choose_signs` method, they should have their own `describe` block.

- [ ] Please wrap them in a `describe` block.


**IMPORTANT!** I have only pointed this out here and above. I have noticed this is the case for some of your tests below. So I won't repeat myself. 

- [ ] Please go through all of the below tests and wrap the tests in a `describe` block that belongs together. Eg., if you have 4 tests for one method, they should be put inside a `describe` block.